lib/foundation/extensions/string_ext.dart

```diff
@@ -1,13 +1,29 @@
 import 'package:easy_localization/easy_localization.dart' as el;
+import 'package:flutter/cupertino.dart';
 
 extension StringExt on String {
   bool get isBlank => trim().isEmpty;
 
-  String get tr {
+  // String get tr {
+  //   try {
+  //     return el.tr(this);
+  //   } catch (e) {
+  //     return this;
+  //   }
+  // }
+}
+
+///
+/// Extension on context to make sure any string that uses [tr] gets translated.
+/// Not using the context won't translate the string if its [tr] is being used in non changing components eg. viewModel
+///
```

Comment:
Let's move this to the function documentation.

Also let's make it more clear that the we are using context to make sure it updates.
```suggestion
///
/// Extension on context to make sure any string that uses [tr] gets updated when translation changes.
/// Not using the context won't translate the string if its [tr] is being used in non changing components eg. viewModel
///
```